### PR TITLE
18 funcionality of the verification of the email and register wit a google

### PR DESCRIPTION
**Pull request by Angel591Diaz**
#### New structure with Xml.

### CODE REVIEW

File: app/src/main/java/com/example/carhive/Data/datasource/remote/Firebase/FirebaseAuthDataSource.kt
Comment (IsmaelMontiel710) [inline app/src/main/java/com/example/carhive/Data/datasource/remote/Firebase/FirebaseAuthDataSource.kt:1]: comments in spanish

**Review by IsmaelMontiel710 — CHANGES_REQUESTED** [review]
the comments in englis please

File: app/src/main/java/com/example/carhive/Data/datasource/remote/Firebase/FirebaseAuthDataSource.kt
Comment (Angel591Diaz) [reply app/src/main/java/com/example/carhive/Data/datasource/remote/Firebase/FirebaseAuthDataSource.kt:1]: No

**Review by MeganPegueros — APPROVED** [review]

File: app/src/main/java/com/example/carhive/Data/datasource/remote/Firebase/FirebaseAuthDataSource.kt
Comment (IsmaelMontiel710) [reply app/src/main/java/com/example/carhive/Data/datasource/remote/Firebase/FirebaseAuthDataSource.kt:1]: yes

Comment (Angel591Diaz) [reply app/src/main/java/com/example/carhive/Data/datasource/remote/Firebase/FirebaseAuthDataSource.kt:1]: No

Comment (Angel591Diaz) [reply app/src/main/java/com/example/carhive/Data/datasource/remote/Firebase/FirebaseAuthDataSource.kt:1]: It is further delaying the delivery for the design team, tomorrow I will comment on it properly

**Review by Uli19 — APPROVED** [review]
Zy

**Review by Gavalencia12 — APPROVED** [review]
The only convenient thing I see are the comments in Spanish, but it doesn't affect the structure of the code. The merge can be done.

**Pull request merged**